third-party/realdds/doc/discovery.md

```diff
@@ -83,6 +93,7 @@ When it is expected that a server will go offline, the server can elect to send
 
 ```JSON
 {
+  "topic-root": "realsense/D405_123622270732",
```

Comment:
Mandatory now? line 103 says no

Reply:
Yes, root is mandatory and cannot change. If it does, it's a different device.

Reply:
I've changed line 103:
>No fields other than the root are necessary with `stopping`.

Reply:
Also changed line 121:
>It is important that the `topic-root` stays the **same as a non-recovery device**: otherwise they are considered different devices.